prow/cmd/peribolos/README.md

```diff
@@ -46,6 +46,9 @@ orgs:
         - anne
         maintainers:
         - jane
+        repos: # Ensure the team has the following permissions levels on repos in the org
+          some-repo: admin
```

Comment:
Once we have repo management I feel like this would be better to scope under the repo.

There's no guarantee that a `some-repo` exists (although I guess one could also argue that there's no guarantee that `some-team` exists)

Reply:
I thought about this, too, but:

1. GitHub's API works through the team, so I figured it's best if we mirror it for code complexity
2. Running on repo, not on team, removes the nice property for dumps above and will make honoring the flag to ignore teams harder
3. Running per repo also adds a substantial API burden to list repo state through teams